app/services/state_file/state_information_service.rb

```diff
@@ -71,8 +72,9 @@ def state_intake_class_names
         submission_builder_class: SubmissionBuilder::Ty2022::States::Az::AzReturnXml,
         survey_link: "https://codeforamerica.co1.qualtrics.com/jfe/form/SV_7UTycCvS3UEokey",
         tax_payment_info_url: "https://azdor.gov/making-payments-late-payments-and-filing-extensions",
-        tax_payment_url: "AZTaxes.gov",
+        tax_payment_url: "azdor.gov/make-payment-online",
```

Comment:
I'm not sure how this ended up diverging, but going forward I think we need to be consistent about having `https://` here in the config to make sure it isn't duplicated later. (I think it had to do with sometimes trying to display a "pretty" version of the url)